### PR TITLE
helpers for integrating computed variables

### PR DESCRIPTION
**Pull request by d-callan**

### CODE REVIEW

**Comment by d-callan** [comment]
theres a chance when i finish up the compute-client branch in the data service that i might decide i want something more here, but for now this is what i know i need.

File: R/methods-VariableMetadata.R [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+  signature = c("x", "y")
+)
+
+#'@export 
+setMethod("merge", signature("VariableMetadataList", "VariableMetadataList"), function(x,y) {
+  veupathUtils::VariableMetadataList(S4Vectors::SimpleList(c(as.list(x), as.list(y))))
+})
+
+#'@export 
+setMethod("merge", signature("VariableSpecList", "VariableSpecList"), function(x,y) {
```
Comment (bobular) [inline R/methods-VariableMetadata.R:21]: Can you make a `VariableMetadataList` from two `VariableSpecList`s?

I can't see how you could.

**Review by bobular — APPROVED** [review]
Looks good, but I have a possibly naive question about the untested (perhaps unused) merge method.

File: R/methods-VariableMetadata.R (hunk 1 above)
Comment (d-callan) [reply R/methods-VariableMetadata.R:21]: Lol no. It's an error

**Pull request merged**